src/base/header.rs

```diff
@@ -775,19 +775,17 @@ impl HeaderSection {
     ///
     /// # Panics
     ///
-    /// This function panics if the the octets slice is shorter than 24
-    /// octets.
+    /// This function panics if the octets slice is shorter than 12 octets.
```

Comment:
Just making a note that this line changes 24 to 12, which change github fails to highlight.

Reply:
That is correct. Not quite sure where the 24 came from, but the code indeed checks for the size of `HeaderSection` which is basically `[u8; 12]`.